From: Michael Tokarev <mjt@tls.msk.ru>
Subject: do not include pc-bios/meson.build from main build as we build all firmware separately

pc-bios/meson.build tries to link various firmware files to the build
directory, but we DFSG-removed them so the build fails to find them.
Just disable entering the subdir entirely since we buile all the
necessary firmware in d/rules anyway.

diff --git a/meson.build b/meson.build
--- a/meson.build
+++ b/meson.build
@@ -2787 +2787 @@ subdir('tools')
-subdir('pc-bios')
+#subdir('pc-bios')
